sklearn/feature_extraction/tests/test_text.py

```diff
@@ -1099,6 +1099,7 @@ def test_vectorizer_string_object_as_input(Vectorizer):
     assert_raise_message(
             ValueError, message, vec.fit_transform, "hello world!")
     assert_raise_message(ValueError, message, vec.fit, "hello world!")
+    vec.fit(["some text", "some other text"])
```

Comment:
Shall we have a check for the `NotFittedError` expliecitely?

Reply:
The think is: we do not really care about which of if `NotFittedError` or `ValueError` is raised first when calling transform on unfitted estimator with invalid input.

Reply:
OK I see